src/battle_script_commands.c

```diff
@@ -8959,6 +8994,101 @@ static void Cmd_various(void)
         gFieldStatuses &= ~STATUS_FIELD_TERRAIN_ANY;    // remove the terrain
         TryToRevertMimicry(); // restore the types of Pokémon with Mimicry
         break;
+    case VARIOUS_JUMP_IF_UNDER_200:
+        // If the Pokemon is less than 200 kg, or weighing less than 441 lbs, then Sky Drop will work. Otherwise, it will fail.
+        if (GetPokedexHeightWeight(SpeciesToNationalPokedexNum(gBattleMons[gBattlerTarget].species), 1) < 441)
+            gBattlescriptCurrInstr = T1_READ_PTR(gBattlescriptCurrInstr + 3);
+        else
+            gBattlescriptCurrInstr += 7;
+        return;
+    case VARIOUS_SET_SKY_DROP:
+        gStatuses3[gBattlerTarget] |= (STATUS3_SKY_DROPPED | STATUS3_ON_AIR);
+        /* skyDropTargets holds the information of who is the attacker and the target of Sky Drop. 
+           This is needed in the case that multiple Pokemon use Sky Drop in the same turn or if
+           the target of a Sky Drop faints while in the air.*/
+        if (gBattleStruct->skyDropTargets[0] == 0xFF)
```

Comment:
I'm confused about these `skyDropTargets` indices. are they not the battler positions? shouldnt it be something long the lines of:
```
gBattleStruct->skyDropTargets[gBattlerTarget] = gBattlerAttacker;
gBattleStruct->skyDropTargets[gBattlerAttacker] = gBattlerTarget;
```

Reply:
They are not the battler positions. `skyDropTargets[0]` holds the battler position of the Pokémon that is currently holding another Pokémon in Sky Drop. This is the attacker of Sky Drop. That other Pokémon's battler position, a.k.a the target of a Sky Drop, is held in `skyDropTargets[1]`. The same can be said for indices 2 and 3, respectively, in case of a double battle where two instances of Sky Drop are occurring at the same time. This storage allows for the identification of both the attacker and the target of a particular instance of Sky Drop.

In short, the Pokémon at the battler position stored in index 0/2 is currently holding the Pokémon at the battler position stored in index 1/3 in Sky Drop, respectively.

Reply:
This system is very convoluted. If you store the sky drop target as I suggested then you then have all the information you need to check the relevant sky drop pairs, even when two sky drops are simultaneously active, in a significantly more readable way.

e.g. consider the accuracy check. You can make it: `|| (!(gBattleMoves[move].flags & (FLAG_DMG_IN_AIR | FLAG_DMG_2X_IN_AIR)) && gStatuses3[gBattlerTarget] & STATUS3_ON_AIR
        && gBattleStruct->skyDropTarget[gBattlerAttacker] != gBattlerTarget)` instead of the long check you have now.

it also makes the set and clear sky drop various commands trivial.